### PR TITLE
LRQA-54501 | Master

### PR DESCRIPTION
**Pull request by tainguyen100**

### CODE REVIEW

**Comment by barrosdennys** [comment]
Just started reviewing :)

**Comment by barrosdennys** [comment]
Just started reviewing :)

File: modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase [hunk 1]
```diff
@@ -1579,6 +1661,31 @@ definition {
...
+
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "Custom Object Title",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		AppBuilderAdmin.createFormViewAndTableView(
+			fieldType = "Text Field",
+			formViewName = "My Form View 1",
+			tableViewName = "My Table View 1");
```
Comment (barrosdennys) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase]: @tainguyen100 
this test is failing while performing this step. Probably related to "Text Field" change. Would you please take a look?

**Comment by barrosdennys** [comment]
Hi @tainguyen100 ,
Thank you very much for sending this test for me to review. Good job!

I have only a few concerns:

1) Is there any specific reason that you added @ignore="true" in the tests?
2) Text Field column is now Text. Please update your test case to cover this change.

**Comment by tainguyen100** [comment]
@barrosdennys Thanks for reviewing Dennys! Didn't realize the test I used had `@ignore` in it, I'll remove that and update "Text Field" to "Text"

**Pull request closed without merging**